nailgun/entities.py

```diff
@@ -6102,7 +6110,7 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
         """
         if attrs is None:
             attrs = self.read_json()
-        return attrs
+        return attrs['registration_command']
```

Comment:
Returning only `registration_command` results will disturb the existing tests of this component. Have you made changes in those tests as well ? 

Or there is no impact ?

Reply:
Yes, already updated https://github.com/SatelliteQE/robottelo/pull/10458/files#diff-3b8b535cc31521aaa9ed258abb8f1877623d212c7402ded2e3362551d774eec7L75-R76